docs/manuals/source/Appendix/DisasterRecoveryUsingBareos.rst

```diff
@@ -76,25 +75,55 @@ Relax-and-Recover (ReaR) is quite easy to use with Bareos.
 Installation
 ^^^^^^^^^^^^
 
-Bareos is a supported backend for ReaR >= 1.15. To use the :strong:`BAREOS_CLIENT` option, ReaR >= 1.17 is required. If ReaR >= 1.17 is not part of your distribution, check the `download section on the
-ReaR website <http://relax-and-recover.org/download/>`_.
+For Bareos >= 22, ReaR > 2.7 is required.
+If ReaR > 2.7 is not part of your distribution, check the `download section on the
+ReaR website <https://relax-and-recover.org/download/>`_.
+The current (2024) stable release of ReaR is 2.7,
+therefore until a newer version is released,
+a ReaR snapshot package must be used,
+see :ref:`section-rear-snapshot-package-installation`.
 
 Configuration
 ^^^^^^^^^^^^^
 
-Assuming you have a working Bareos configuration on the system you want to protect with ReaR and Bareos references this system by the name :strong:`bareosclient-fd`, the only configuration for ReaR is:
+For ReaR to work with Bareos, the following packages must be installed:
+
+* **rear** > 2.7 (currently only via a ReaR snapshot package)
+* **bareos-fd**
+* **bareos-bconsole**
+
+Make sure you have a working |fd| configuration on the system and already did a backup of all disks you want to restore.
+You also need the |bconsole| installed and configured,
+so that it can connect to the |dir|.
+Ideally the |bconsole| is configured in a way that it can only access this client,
+see :ref:`section-named-console-for-rear`.
+
+To configure the Bareos backend, ReaR uses the variables
```

Comment:
Not sure if you have this, but it might be worth explaining that the fileset used for ReaR should be a full backup of all files and not a partial data-only backup.

Reply:
This has only be mentioned in the introduction, but making it more explicit is probably better. Added it now.